packages/cli/src/lib/setup/setupInstall.ts

```diff
@@ -58,14 +56,14 @@ interface CreateInstanceOptions {
     port?: number;
 }
 
-class Install {
+export class Install {
     private unsafePermAlways: string[];
     private readonly isRootOnUnix: boolean;
     private readonly objects: ObjectsRedisClient;
     private readonly states: StatesRedisClient;
-    private readonly processExit: (exitCode?: number) => void;
-    private readonly getRepository: any;
-    private readonly params: Record<string, any>;
+    private readonly processExit: CLIInstallOptions['processExit'];
+    private readonly getRepository: CLIInstallOptions['getRepository'];
+    private readonly params: CLIInstallOptions['params'];
```

Comment:
I think it would be cleaner to extract those types into their own `type` definition and use them here and in `CLIInstallOptions`

Reply:
Fine for me, I haven't seen that much TS projects to be sure what good style is here.